network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_index_callout_box.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags static i18n %}
+
+<div class="tw-h-full tw-flex tw-flex-col tw-justify-between tw-p-[36px] tw-border tw-border-black tw-shadow-[10px_10px_rgba(0,0,0)]">
+  <div id="callout-box-body" class="tw-mb-[4rem]">
+        <div class="tw-flex">
+          {% if value.related_topics %}
+            <div class="tw-mb-3">         
+              {% include '../fragments/blog_topics.html' with topics=value.related_topics %}
+            </div>
+          {% endif %}
+          {% if value.show_icon %}
+            <img src="{% static "_images/headphones-icon.svg" %}" alt="{% trans "audio" context "icon description for screen readers" %}" class="tw-h-4 tw-w-4 tw-mr-2 tw-my-1 tw-mb-3">
+          {% endif %}
+        </div>
+        <h1 class="tw-text-[40px] tw-mb-2">{{ value.title }}</h1>
```

Comment:
**Issue** There should never be more than one `h1` element on a given page. We already use this for the page title. 

Since this element is only used on the index page and not nested, it should use an `h2` to not break the semantic structure of the page. Styling should be applied via CSS. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#multiple_h1_elements_on_one_page

Reply:
Updated!